karspexet/ticket/models.py

```diff
@@ -90,3 +90,6 @@ class PricingModel(models.Model):
 
     def price_for(self, ticket_type):
         return int(self.prices[ticket_type])
+
+    def __repr__(self):
+        return f"PricingModel id:{self.id} prices:{self.prices} pricing_group:{self.seating_group_id}-{self.seating_group.name} valid_from:{self.valid_from}"
```

Comment:
I just looked and the standard way to represent a model object is `<User maxno>`. That, or follow the pythonic guideline of being able to evaluate the resulting string in an appropriate context and get the same object back. E.g. `Model(key=word, args=[])`. Also the long line kills me a bit on the inside, but what can you do?